IdentityCore/src/IdentityCore.pch

```diff
@@ -36,4 +36,8 @@
 #import <AppKit/AppKit.h>
 #endif
 
+#if __has_include("../../../../../aad_overrides.h")
```

Comment:
If you can be extra nice, you could add a comment for future generation of MSAL devs on why this is necessary :)